pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -781,6 +782,24 @@ func newKubeVirtTemplateValidatorForCR(cr *hcov1alpha1.HyperConverged, namespace
 	}
 }
 
+func newIMSConfigForCR(cr *hcov1alpha1.HyperConverged, namespace string) *corev1.ConfigMap {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "v2v-vmware",
+			Labels:    labels,
+			Namespace: namespace,
+		},
+		Data: map[string]string{
+			"v2v-conversion-image":              "registry.redhat.io/container-native-virtualization/kubevirt-v2v-conversion:v2.0.0",
+			"kubevirt-vmware-image":             "registry.redhat.io/container-native-virtualization/kubevirt-vmware:v2.0.0",
```

Comment:
@djzager is there somewhere good to set these as a const or param, or is this the 'best' way to go.

I also, have a question out about whether there will be a floating tag for 2.1.0... I don't see any builds in brew yet.

Reply:
> @djzager is there somewhere good to set these as a const or param, or is this the 'best' way to go.

I think the "best" way to do this is to source it from an environment variable on the deployment of the HCO. My hope would be there is an upstream variant of these images.


> I also, have a question out about whether there will be a floating tag for 2.1.0... I don't see any builds in brew yet.

I think @rthallisey is the best resource for you with respect to brew builds/floating tags. This may be something that needs to be included in the build-pipeline that has been being worked on.

Reply:
> I think the "best" way to do this is to source it from an environment variable on the deployment of the HCO. My hope would be there is an upstream variant of these images.

I say this because then it can be added to the CSV and it's controllable in the build process what values will go into the config map that is created.